data/data/bootstrap/baremetal/files/usr/local/bin/master-bmh-update.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+. /usr/local/bin/release-image.sh
```

Comment:
The last shellcheck warning is related to this.

```suggestion
# shellcheck disable=SC1091
. /usr/local/bin/release-image.sh
```

Reply:
What's strange to me is that it's the same as the other scripts and they don't have it.. maybe it's new.

Reply:
I saw that too - maybe one of the bash settings at the top of those scripts?